pyrokinetics/kinetics/Kinetics.py

```diff
@@ -63,13 +62,13 @@ def __init__(
 
     @property
     def kinetics_type(self):
-        return _kinetics_type
+        return self._kinetics_type
 
     @kinetics_type.setter
     def kinetics_type(self, value):
         if value not in self.supported_kinetics_types:
             raise ValueError(f"Kinetics type {value} is not currently supported.")
-        _kinetics_type = value
+        self._kinetics_type = value
```

Comment:
Apologies for not catching that bug! I'll include unit tests for it in the next pull request.